### ISSUE TITLE
AMD H.264 issue - Regardless of options in StaxRip, output bitrate is around 5000~

### ISSUE THREAD

**Issue by nulledone** [rigaya/VCEEnc#26]
I think this one might also be about 5000 series cards. I've noticed that StaxRip defaults VBV Bufsize to 5000 kbps, however when I changed that value to 20001 output bitrate is still 5000.

Here is the log, hope it helps. 
[2021-04-14 - 14.45.59 - 20210414_000546_staxrip.log](https://github.com/rigaya/VCEEnc/files/6310607/2021-04-14.-.14.45.59.-.20210414_000546_staxrip.log)

Previous log file submitted does not have this issue because it uses variable bitrate rather than CQP, I can confirm that this only occurs while using CQP (tried variable and constant bitrate options)

Also another log in which I've set CQP as 1:2:3
[2021-04-14 - 14.58.34 - 20210414_000546_staxrip.log](https://github.com/rigaya/VCEEnc/files/6310631/2021-04-14.-.14.58.34.-.20210414_000546_staxrip.log)

I've also manually started the command below using powershell to see if this issue is about avs&staxrip but the output is same;
C:\Z\PROGRAMS\StaxRip-v2.4.0-x64\Apps\Encoders\VCEEnc\VCEEncC64.exe --cqp 40:50:50 -i Z:\-\Media\Video\Game\20210414_000546.mp4 -o Z:\-\TempCache\Staxrip\20210414_000546_temp\20210414_000546_out2.h264

```
General
Complete name               : cqp1-2-3.h264
Format                      : AVC
Format/Info                 : Advanced Video Codec
File size                   : 18.9 MiB

Video
Format                      : AVC
Format/Info                 : Advanced Video Codec
Format profile              : High@L5.1
Format settings             : CABAC / 1 Ref Frames
Format settings, CABAC      : Yes
Format settings, Reference  : 1 frame
Format settings, GOP        : M=1, N=30
Width                       : 1 920 pixels
Height                      : 1 080 pixels
Display aspect ratio        : 16:9
Frame rate                  : 30.000 FPS
Standard                    : Component
Color space                 : YUV
Chroma subsampling          : 4:2:0
Bit depth                   : 8 bits
Scan type                   : Progressive
Color range                 : Limited
```

```
General
Complete name               : cqp40-50-50.h264
Format                      : AVC
Format/Info                 : Advanced Video Codec
File size                   : 18.9 MiB

Video
Format                      : AVC
Format/Info                 : Advanced Video Codec
Format profile              : High@L5.1
Format settings             : CABAC / 1 Ref Frames
Format settings, CABAC      : Yes
Format settings, Reference  : 1 frame
**Format settings, GOP        : M=1, N=30**
Width                       : 1 920 pixels
Height                      : 1 080 pixels
Display aspect ratio        : 16:9
Frame rate                  : 30.000 FPS
Standard                    : Component
Color space                 : YUV
Chroma subsampling          : 4:2:0
Bit depth                   : 8 bits
Scan type                   : Progressive
Color range                 : Limited
```

This image you've submitted might have also had this issue since output sizes are different (unless you've used different settings)

![](https://user-images.githubusercontent.com/815947/114552320-7022e400-9c9f-11eb-902b-9e46ddadfc50.png) ![](https://user-images.githubusercontent.com/815947/114553163-503ff000-9ca0-11eb-86ce-bb0b5ce08663.png)

I notice in that image (and on my output) that there's a one extra line, **Format settings, GOP        : M=1, N=30**

**Comment by 44vince44** [comment]
This is not a problem with VCEEnc, in your script, you set
`--vbv-bufsize 20001 '
But this is not how you set the output bitrate
this is how `--max-bitrate 20001`
and of course you need to increase the bufsize accordingly.

**Comment by rigaya** [comment]
I was able to fix this issue too in [VCEEnc 6.10](https://github.com/rigaya/VCEEnc/releases/tag/6.10).

The output is now changing depending on the cqp value, and there is no extra "Format settings, GOP : M=1, N=30" line anymore.

**Comment by nulledone** [comment]
Seems to be working well, thanks!
